stglib/rsk/nc2waves.py

```diff
@@ -47,13 +47,157 @@ def nc_to_waves(nc_filename):
 
     nc_filename = ds.attrs["filename"] + "s-a.nc"
 
-    for var in ds.data_vars:
-        if "time" not in var:
-            # cast as float32
-            ds = utils.set_var_dtype(ds, var)
-
     ds.to_netcdf(nc_filename, unlimited_dims=["time"])
     utils.check_compliance(nc_filename, conventions=ds.attrs["Conventions"])
     print("Done writing netCDF file", nc_filename)
 
     return ds
+
+
+def ds_add_attrs(ds):
```

Comment:
It looks like this was copy-pasted from somewhere. Can the redundant code be reduced? We shouldn't have any reference to epic_time in new code.